components/kyma-environment-broker/internal/event/application_broker.go

```diff
@@ -0,0 +1,48 @@
+package event
+
+import (
+	"context"
+	"reflect"
+	"sync"
+)
+
+type Handler = func(ctx context.Context, ev interface{}) error
+
+type Publisher interface {
+	Publish(ctx context.Context, event interface{})
+}
+
+// ApplicationEventBroker implements a simple event broker which allows to send event across the application.
+type ApplicationEventBroker struct {
+	mu sync.Mutex
+
+	handlers map[reflect.Type][]Handler
+}
+
+func NewApplicationEventBroker() *ApplicationEventBroker {
```

Comment:
what about sth like `NewPubSub`? just to avoid words like `application`, `broker` which can be misleading because of our domain (we already have Application, we already have broker). 

IMO `NewPubSub` exactly says what will be returned

Reply:
done